modules/custom/az_core/az_core.module

```diff
@@ -22,3 +25,37 @@ function az_core_help($route_name, RouteMatchInterface $route_match) {
     default:
   }
 }
+
+/**
+ * Implements hook_pathauto_pattern_alter().
```

Comment:
This section generally comes from what was is in UAQS. The $alias part was my attempt at doing what pathauto does in [PathautoGenerator](https://git.drupalcode.org/project/pathauto/-/blob/8.x-1.x/src/PathautoGenerator.php#L223). Review of this part would be helpful, especially if something should be handled differently.